site/en/blog/cross-origin-isolated-hr-timers/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Aligning timers with cross origin isolation restrictions
+authors:
+  - yoavweiss
+  - agektmr
+description: Starting in Chrome 91, the resolution of explicit timers will be restricted to 100 microseconds across platforms without cross-origin isolation.
+date: 2021-04-15
+hero: 'image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/mnoYgEv2E9L6eSThY9G4.jpg'
+alt: A stopwatch
+---
+
+Before version 91,
+[timer resolutions](https://www.w3.org/TR/hr-time/)
+on Chrome were restricted to 5 microseconds on desktop,
+where [site-isolation](https://www.chromium.org/Home/chromium-security/site-isolation) is enabled,
+and to 100 microseconds on Android, where it's not.
+
+Starting from version 91,
+Chrome will be restricting the resolution of explicit [timers](https://www.w3.org/TR/hr-time/)
+(`performance.now()`, `performance.timeOrigin()`,
+and other performance APIs that expose `DOMHighResTimestamps`) to 100 microseconds across platforms.
+By enabling [cross-origin isolation](https://web.dev/coop-coep/),
+websites can relax the restriction to 5 microseconds regardless of platform.
+
+{% Aside %}
+A microsecond is one millionth of a second (0.000001 seconds). Therefore, 5 microseconds is 0.000005 seconds.
+{% endAside %}
+
+Gaining more powerful capabilities by enabling cross-origin isolation is
+[a similar approach posed on `SharedArrayBuffer`](https://developer.chrome.com/blog/enabling-shared-array-buffer/).
+Cross-origin isolation is a state where a web page is isolated from other origins except ones that opted-in.
+
+## Do I need to do something about it?
+
+Probably not. `performance.now()` has been limited to significantly more coarse resolutions in other browsers
+(e.g. 1ms = 0.001 seconds),
```

Comment:
I'd change `1ms` to `1 millisecond` so the difference with `1 microsecond` is more explicit.